lib/rules/web/admin_console/4.4/monitoring_alerts.xyaml

```diff
@@ -0,0 +1,112 @@
+click_icon_index_button:
+  element:
+    selector:
+      xpath: (//button[@aria-label='<aria_label_text>'])[<index>]
+    op: click
+set_placeholder_input_value:
+  element:
+    selector:
+      xpath: (//input[@placeholder='<placeholder_text>'])[<index>]
+    op: send_keys <input_value>
+    type: input
+clear_label_input_value:
+  elements:
+  - selector:
+      xpath: //label[text()='<label_text>']/..//input
+    op: clear
+    type: input
+set_label_input_value:
```

Comment:
I think it can be replaced with action `set_label_input`

Reply:
can't be change, as there is one with same name and different definition in the common UI